ios/Exponent/Versioned/Core/UniversalModules/EXScopedFontLoader.m

```diff
@@ -0,0 +1,17 @@
+// Copyright © 2019-present 650 Industries. All rights reserved.
+
+#if __has_include(<EXFont/EXFontLoader.h>)
+#import "EXScopedFontLoader.h"
+#import "EXConstantsBinding.h"
+#import <UMConstantsInterface/UMConstantsInterface.h>
+
+static NSString *expoPrefix = @"ExpoFont-";
```

Comment:
const

Reply:
good call, i just copied the existing name `expoFont` over, but better to use the `EX*` convention